dnf/rpm/__init__.py

```diff
@@ -23,7 +23,7 @@
 from dnf.pycomp import is_py3bytes
 import dnf.const
 import dnf.exceptions
-import rpm
+import rpm  # :api (ansible uses dnf.rpm.rpm)
```

Comment:
I am sorry, but this is wrong, we have to ask them to not do it

Reply:
Yes it is. I'll send PR to ansible to stop importing rpm via dnf. You are right we should not mark this as an API, but we should leave at least some comment here that ansible is actually using it.